app/src/main/java/com/rootstrap/android/di/DI.kt

```diff
@@ -72,6 +77,7 @@ val dataModule = module {
     single { Prefs(get()) }
     single { UserRepository(get()) }
     single { ProductRepository() }
+    single { UserCartRepository() }
```

Comment:
In the future, be careful with this file. It seems that is getting bigger with the size of the app.